src/redux/store.js

```diff
@@ -0,0 +1,13 @@
+import { createStore, combineReducers } from '@reduxjs/toolkit';
```

Comment:
- [ ] As stated in the requirements, let's kindly utilize configureStore to create a reducer. We will require to, first of all, import it from @reduxjs/toolkit